libs/LedKit/include/LEDAnimation.h

```diff
@@ -15,6 +17,14 @@ namespace leka::led::animation::utils {
 	return leka::utils::math::map(step, uint8_t {0}, step_max, 0.F, 1.F);
 }
 
+[[nodiscard]] inline auto sinPeriod(uint8_t period, uint8_t step, uint8_t offset) -> float
+{
+	auto mapped_offset = leka::utils::math::map(offset, uint8_t {0}, uint8_t {255}, 0.F, float {2 * M_PI});
+	auto x			   = 2 * M_PI * (1.F / static_cast<float>(period)) * step + mapped_offset;
+	auto sinus		   = sin(x);
```

Comment:
```suggestion
	auto sinus		   = std::sin(x);
```

c'est la même chose mais c'est plus sûr